src/main/java/battlestates/LoseBattleState.java

```diff
@@ -0,0 +1,77 @@
+package battlestates;
+
+import interfaces.State;
+import io.InputValidator;
+import io.OutputHandlerImpl;
+import objects.character.Enemy;
+import objects.character.Player;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LoseBattleState implements State {
```

Comment:
If all you are doing before user input is asking a question, I suggest you extend ChoiceInputState
override the postInput() method to do whatever you need to do after the question
this automatically deals with creating the question and inputValidator